index.html

```diff
@@ -19,6 +19,12 @@
     <title>HOME</title>
   </head>
   <body>
+    <img
+      src="./images/hero-beach.png"
+      alt="hero-beach-image"
```

Comment:
@RafaelMariscal, check the `alt` attribute. It should be a description that It will appear if the image fails to load. My suggestion:
`<img
      src="./images/hero-beach.png"
      alt="Borda de uma praia no por do sol"
      class="main-image"
    />`

Reply:
@lucas-segundo  Ok!

Reply:
@RafaelMariscal, good job! Thank you.